app/gui/src/dashboard/data/serviceCredentials/GoogleCredentialsForm.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * Dialog for a Google credential.
+ * Remember to ensure this component is added to `CREDENTIAL_INFOS` in `constants.ts`.
+ */
+
+import { Checkbox, Form, Input } from '#/components/AriaComponents'
+import { useToastAndLog } from '#/hooks/toastAndLogHooks'
+import { useText } from '#/providers/TextProvider'
+import { CredentialsFormButtons } from './CredentialsFormButtons'
+import * as google from './google'
+import type { CredentialFormProps } from './types'
+
+/** Dialog for a Google credential. */
+export function GoogleCredentialsForm(props: CredentialFormProps) {
+  const { createCredentials } = props
+  const { getText } = useText()
+  const toastAndLog = useToastAndLog()
+
+  const form = Form.useForm({
```

Comment:
It's better to inline form here: 

` <Form schema={google.FORM_SCHEMA} method="dialog" className="w-full" onSubmit={....}>`

Reply:
Same true for the other providers